src/styles.css

```
@@ -1,39 +1,130 @@
+.sources-error { align-items: center; background: #fff4ef; border-left: 3px solid #d36543; display: flex; gap: 14px; padding: 14px 16px; }
+.sources-error strong { color: #a64c2e; }.sources-error span { color: #765348; flex: 1; }
+.sources-error button { background: transparent; border-color: #d36543; color: #a64c2e; min-height: 32px; }
+
+.scope-app { display: flex; flex-direction: column; }
+.scope-app .workspace-header { border-bottom: 1px solid #dbe6e0; margin-bottom: 0; padding-bottom: 23px; }
+.compact-source-form { align-items: center; display: flex; gap: 8px; margin: 0; }
+.compact-source-field { display: block; }
+.compact-source-field input { min-height: 36px; width: 190px; }
+.source-name-field input { width: 132px; }
+.compact-source-form button { min-height: 36px; white-space: nowrap; }
+.visually-hidden { height: 1px; margin: -1px; overflow: hidden; position: absolute; width: 1px; clip: rect(0 0 0 0); }
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)